python/pyspark/sql/tests.py

```diff
@@ -351,7 +351,7 @@ def tearDown(self):
         super(SQLTests, self).tearDown()
 
         # tear down test_bucketed_write state
-        self.spark.sql("DROP TABLE IF EXISTS pyspark_bucket")
+        self.spark.sql("DROP DATABASE IF EXISTS some_db CASCADE")
```

Comment:
why do we need to drop the database now?

Reply:
test_current_database,test_list_databases,test_list_tables,test_list_functions and test_list_columns all these test case create the database and does not drop it,so the folder will be present which result in the test case failures

Reply:
Could you update the corresponding test cases instead of dropping the database here?